erpnext/controllers/selling_controller.py

```diff
@@ -115,7 +115,7 @@ def set_total_in_words(self):
 		from frappe.utils import money_in_words
 
 		if self.meta.get_field("base_in_words"):
-			base_amount = abs(self.base_grand_total
+			base_amount = abs(self.base_grand_total)
```

Comment:
- This is not an issue, if you see its a single line `if-else` condition.
- Can you check the error traceback again what this issue is

Reply:
Sure will check the error traceback again